Sources/NIO/EventLoop.swift

```diff
@@ -584,7 +592,11 @@ extension EventLoop {
     /// - returns: a succeeded `EventLoopFuture`.
     @inlinable
     public func makeSucceededFuture<Success>(_ value: Success, file: StaticString = #file, line: UInt = #line) -> EventLoopFuture<Success> {
-        return EventLoopFuture<Success>(eventLoop: self, value: value, file: file, line: line)
+        if /*!_isDebugAssertConfiguration() && */ (Success.self == Void.self) {
```

Comment:
Comment should be removed, this can always run like this. We don't need `file`/`line` for  succeeded future because the `file`/`line` is only relevant for the debugging if the promise isn't fulfilled (which can't be the case here).